[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/drbob-richardson/stat220/blob/main/Assignments/Stat_220_HW2.ipynb)

**Problem 1** A baseball player has a batting average of .240 against right handed pitchers and .375 against left handed pitchers. A batting average can be thought of as the probability they will get a hit.

Part a. What is the probability the baseball player gets 3 out of 4 hits against a right handed pitcher?

In [10]:
import math

n = 4
k = 3
p = .240

coeff = math.comb(n, k)
coeff * (p ** k) * ((1 - p) ** (n - k))

0.04202495999999999

Part b. What is the probability the baseball gets 3 out of 4 hits against a left handed pitcher.

In [9]:
import math

n = 4
k = 3
p = .375

coeff = math.comb(n, k)
coeff * (p ** k) * ((1 - p) ** (n - k))

0.1318359375

Part c. The baseball player gets 3 out of 4 hits against a randomly chosen pitcher. Based on these results, what would you assume about what hand the pitcher uses?

According to the results of these binomial distributions it is roughly 3 times more likely the pitcher was left handed (4.20% vs 13.18%).

Part d. Assume that the pitcher is randomly chosen from the team where 80% of the pitchers are right handed. Use Bayes rule to determine the probability the pitcher is right handed given that the batter hits 3 out of 4 pitches.

In [8]:
numerator = .04202495999999999 * .8
denominator = .04202495999999999 * .8 + .1318359375 * .2
numerator / denominator

0.5604527789286491

**Problem 2**  
In Arizona, the long-run probability of rain on any given day of the year is about 5%. A researcher observes that it rained three days in a row and calculates the probability of this event as follows:

$$
Pr(\text{3 days of rain}) = Pr(\text{rain on a day})^3 = 0.05^3 = 0.000125
$$

He concludes that this was an extremely rare event.  

You are asked to critique his calculation.  

**Question:** What incorrect assumption about the *relationship between consecutive days of rain* did the researcher make in this calculation, and why does that make his probability estimate misleading?  



He assumed that days of rain are independent to each other. As in if it rains one day it has no effect on whether it rains the next day or not. When most likely these events are not independent and if it rains on one day there is a greater probability it will rain on consecutive days. Therefore his estimation for the probability of this event occuring is most likely low.

**Problem 3** A stock trader is examining stocks. Let event A be the event that Stock A falls. Let event B be the event that Stock B falls. Assume the following.

*   Pr(A) = 0.4
*   Pr(B) = 0.25
*   Pr(A and B) = 0.15


Part a. What is the probability that either events A or B happen?

In [7]:
.4 + .25 -.15

0.5

Part b. From this, can you make statement as to whether or not events A and B are mutually exclusive?

They are not mutually exclusive because there is a non-zero probabilty given where both A and B can occur.

Part c. What can you say about if events A and B are independent?

A and B would be independent if the Pr(A) * Pr(B) == Pr(A∩B). However Pr(A) * Pr(B) = .4 * .25 = .10 != .15 (Pr(A∩B)). Therefore they are not independent.

**Problem 4** Each time a person shops at a grocery store, the event of catching a cold or some other virus from another shopper is independent from visit to visit and has a constant probability over the year, equal to 0.01.

Part a. In 100 trips to this store over the course of a year, the probability of catching a virus while shopping there is 100(0.01) = 1.0. What is wrong with this reasoning?

You can't add probabilities like this because they don't stack linearly. This problem should be solved with the complement rule. Because they are independent you can multiply the probability of the event not occuring (complement) by itself for however many times the event could occur which would give you the probability of the given event not happening. Then you can subtract it by one to find the probability of it happening.

Part b. Find the correct probability in (a).

In [6]:
1-.99**100

0.6339676587267709

Part c. Determine the expected number of days someone would go to a grocery store before they catch a cold or some other virus.

In [13]:
1/.01

100.0

**Problem 5** A quiz has twelve multiple-choice questions, with four possible answers for each. A student who is completely unprepared randomly guesses the answer for each question. Let Y denote the number of correct responses

Part a. What is the name of the distribution that can be used to determine the number of questionsd guessed correctly? What are the parameters of this distribution?

Binomial Distribution because it has a fixed number of trials and we want to know how many successes. Inputs are number of trials and probability of a success.

Part b. Find the probability that the students gets (i) all 12, (ii) none of the questions correct.

In [14]:
# i
import math

n = 12
k = 12
p = .25

coeff = math.comb(n, k)
coeff * (p ** k) * ((1 - p) ** (n - k))

5.960464477539063e-08

In [15]:
# ii
import math

n = 12
k = 0
p = .25

coeff = math.comb(n, k)
coeff * (p ** k) * ((1 - p) ** (n - k))

0.03167635202407837

Part c. What is the expected value and variance of the number of questions guessed correctly?

In [16]:
12*.25

3.0

In [17]:
12*.25*(1-.25)

2.25